product_docs/docs/pgd/5/node_management/creating_and_joining.mdx

```diff
@@ -53,19 +57,21 @@ that already exists on databases in the PGD group. We recommend that the
 newly joining database be empty except for the BDR extension. However,
 it's important that all required database users and roles are created.
 
-Optionally, you can skip the schema synchronization using the `synchronize_structure`
-parameter of the [`bdr.join_node_group`](/pgd/latest/reference/nodes-management-interfaces#bdrjoin_node_group) function. In this case, the schema must
-already exist on the newly joining node.
+Optionally, you can skip the schema synchronization using the
+`synchronize_structure` parameter of the
+[`bdr.join_node_group`](/pgd/latest/reference/nodes-management-interfaces#bdrjoin_node_group)
+function. In this case, the schema must already exist on the newly joining node.
 
 We recommend that you select the source node that has the best connection (the
-closest) as the source node for joining. Doing so lowers the time
-needed for the join to finish.
+closest) as the source node for joining. Doing so lowers the time needed for the
```

Comment:
Could we be more precise here on which characteristics a "best connection" should have? Low latency? High bandwidth?

Reply:
Revised.